linalg/sparsemat.hpp

```diff
@@ -80,9 +86,28 @@ class SparseMatrix : public AbstractSparseMatrix
    void Destroy();   // Delete all owned data
    void SetEmpty();  // Init all entries with empty values
 
+#ifdef MFEM_USE_CUDA
+   cusparseStatus_t status;
+   cusparseHandle_t handle=0;
+   cusparseMatDescr_t descr=0;
+   mutable size_t bufferSize{0};
+   mutable void *dBuffer = NULL;
+   mutable bool isInit{false};
```

Comment:
A more unique name for `isInit` would be helpful, as this could be confused for other types of initializations.